packages/excalidraw/data/restore.ts

```diff
@@ -387,7 +387,10 @@ export const restoreElement = (
             elbowed: true,
             startBinding: repairBinding(element, element.startBinding),
             endBinding: repairBinding(element, element.endBinding),
-            fixedSegments: element.fixedSegments,
+            fixedSegments:
+              element.fixedSegments?.length && base.points.length >= 4
+                ? element.fixedSegments
+                : null,
```

Comment:
not sure if this holds

Reply:
Yes, because the start and the end point itself can **never** be part of a fixed segment, therefore you need to have at least an extra 2 points in the arrow to have a valid configuration.